src-tauri/src/mod_management/mod.rs

```diff
@@ -12,8 +13,6 @@ use app::GameInstall;
 
 use json5;
 
-use crate::northstar::CORE_MODS;
-
 pub const BLACKLISTED_MODS: [&str; 3] = [
```

Comment:
Would it make sense to move those to constants as well?

Reply:
Done with 3d62be62e79f0f079eaa73e4072836da163da020 and also found one more and moved with 0eab7afe6ce0e2e14674b4a364faa52fdd75c37c ^^